cmd/cli/plugin/package/display_utils.go

```diff
@@ -14,15 +15,15 @@ import (
 )
 
 // DisplayProgress creates an spinner instance; keeps receiving the progress messages in the channel and displays those using the spinner until an error occurs
-func DisplayProgress(initialMsg string, pp *tkgpackagedatamodel.PackageProgress) error {
+func DisplayProgress(initialMsg string, pp *tkgpackagedatamodel.PackageProgress) error { //nolint:gocyclo
 	var (
 		currMsg string
 		s       *spinner.Spinner
 		err     error
 	)
 
 	newSpinner := func() (*spinner.Spinner, error) {
-		s = spinner.New(spinner.CharSets[9], 100*time.Millisecond)
+		s = spinner.New(spinner.CharSets[9], 100*time.Millisecond, spinner.WithWriter(os.Stderr))
```

Comment:
Why are we replacing [default colorized stdout](https://github.com/fatih/color/blob/master/color.go#L27) with os.Stderr?

Reply:
Tried to follow the docs of the spinner component. https://github.com/briandowns/spinner#unix-pipe-and-redirect

Reply:
If I am reading the doc correctly this is recommended for chain, pipe or redirecting the output. Could we verify if this is still needed with tty check?